vizro-core/src/vizro/models/_components/card.py

```diff
@@ -41,5 +60,10 @@ def build(self):
             else text
         )
 
-        card_class = "card-nav" if self.href else ""
-        return dbc.Card(children=card_content, className=card_class)
+        defaults = {
+            "children": card_content,
+            "class_name": "card-nav" if self.href else "",
+        }
+
+        finals = defaults | self.extra
+        return dbc.Card(**finals)
```

Comment:
Let's be consistent between all Vizro components that support `extra` when using `finals` vs `**(defaults | self.extra)`.
```suggestion
        return dbc.Card(**(defaults | self.extra))
```